src/web_app_skeleton/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.8'
+services:
+  lucky:
+    build:
+      context: .
+      dockerfile: docker/development.dockerfile
+    environment:
+      DATABASE_URL: postgres://postgres@postgres:5432/postgres
+    volumes:
+      - type: bind
+        source: .
+        target: /app
+      - type: volume
+        source: node_modules
+        target: /app/node_modules
+      - type: volume
+        source: shards_lib
+        target: /app/lib
+    depends_on:
+      - postgres
+
+    entrypoint: ["docker/dev_entrypoint.sh"]
+
+  postgres:
```

Comment:
One thing we could include for all is to boot this postgres instance (and auto-connect) in the normal `Procfile`, since then we can make lucky auto connect to dev DB's

Reply:
I'm not sure what you mean.